packages/plugin-vectorizer/PLAYGROUND.md

```diff
@@ -0,0 +1,180 @@
+
+Define feature flags 
+
+Feature Enabled 
+Feature Permissions 
+
+
+
+
+
```

Comment:
This file seems like a mental note. Not sure I'd like this to be merged, could be an issue or extracted into another PR